README.md

```diff
@@ -1,89 +1,94 @@
 ![Paprika logo](https://user-images.githubusercontent.com/10501940/52080175-07327400-254c-11e9-9748-7a00f93a13a8.png)
 
-# Paprika
+<h1 align="center">Paprika</h1>
+<p align="center">Paprika is a react component library of the <a href="https://design.wegalvanize.com">Starling Design System</a>.</p>
+<div align="center">
+<img alt="react version" src="https://img.shields.io/badge/react-v16.8.2-green.svg">
+<img alt="node version" src="https://img.shields.io/badge/node-v10.0.0%2B-brightgreen.svg">
+<img alt="yarn version" src="https://img.shields.io/badge/yarn-v1.12.3%2B-yellowgreen.svg">
+<img alt="GitHub issues" src="https://img.shields.io/github/issues/acl-services/paprika">
+<img alt="GitHub" src="https://img.shields.io/github/license/acl-services/paprika">
+</div>
+<br>
 
-## Getting started
+Paprika is a collection of reusable components that are versioned and downloaded individually which is built and maintained by [Galvanize](https://www.wegalvanize.com).
 
-Our list of components can be find for now at [npm](https://www.npmjs.com/org/paprika).
+[Storybook](https://paprika.highbond.com) |
+[Npm](https://www.npmjs.com/org/paprika) |
+[Airtable](https://airtable.com/shrkJwkvtbgc3FT22)
 
-Paprika has a `peerDependecy` of [styled-components](https://www.styled-components.com/docs/basics#installation) so make sure to install it as a dependency in your project.
+## Installation and usage
 
-All paprika components can be installed as:
+### Prerequisites
 
-```sh
-  npm install @paprika/ComponentToConsume
-```
-
-or
+Paprika has a `peerDependecy` of `styled-components` so make sure to install it as a dependency in your project.
 
 ```sh
-  yarn add @paprika/ComponentToConsume
+yarn add styled-components
 ```
 
----
-
-## Contributors guide.
+- Node v10.0.0+
+- Yarn v1.12.3+
 
-### Basic installation
+### Example of installing a component
 
-Follow the next steps to start coding on paprika
+The following is an example of the button component, but all other components will be installed in a similiar way.
 
-1.- Install dependencies
+1. Installing the button package:
 
 ```sh
-$ yarn
+yarn add @paprika/button
 ```
 
-2.- Link packages with Lerna
+2. In your project:
 
-```sh
-$ npx lerna bootstrap
+```js
+import React from "react";
+import Button from "@paprika/button";
+
+export default <Button>Hello</Button>;
```

Comment:
This example makes no sense 😄 Let's at least say
```suggestion
export default () => <Button>Hello</Button>;
```

Reply:
This is what happens when doing documentation while sick looool